backend/index.js

```diff
@@ -14,4 +18,12 @@ app.listen(port,()=>{
     console.log('API is up and running')
 })
 
+//Quote
+app.get('/quote', (req, res) => {
+    finnhubClient.quote("AAPL", (error, data, response) => {
```

Comment:
Maybe add in a URL parameter so it's easier to integrate with the front end: `var company = req.query.company;`

Reply:
I agree with Sunny here.